app/Http/Requests/JournalRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class JournalRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return false;
```

Comment:
This needs to return true; 
`If the authorize method returns false, a HTTP response with a 403 status code will automatically be returned and your controller method will not execute.`